opentracing/README

```diff
@@ -0,0 +1,3 @@
+"The Essential Tool for Testing in Production: Distributed Tracing" by Priyanka Sharma
```

Comment:
best to call the file README.md, otherwise it renders without markup.

Reply:
@yurishkuro it's already updated by a separate commit - https://github.com/cncf/presentations/blob/master/opentracing/README.md